manifests/base/params.env

```diff
@@ -1,45 +1,45 @@
-odh-workbench-jupyter-minimal-cpu-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-minimal-cuda-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-minimal-rocm-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-datascience-cpu-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-pytorch-cuda-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-pytorch-rocm-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-tensorflow-cuda-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-tensorflow-rocm-py311-ubi9-n-1=dummy
-odh-workbench-jupyter-trustyai-cpu-py311-ubi9-n-1=dummy
-odh-workbench-codeserver-datascience-cpu-py311-ubi9-n-1=dummy
+odh-workbench-jupyter-minimal-cpu-py311-ubi9-n-1=quay.io/modh/odh-minimal-notebook-container@sha256:f9ea0155a47457294a158bd3bef2ba5ab9ed9463e296a2f7304a0f912f0eff5c
```

Comment:
this should not be /modh, see https://issues.redhat.com/browse/RHAIENG-1086?focusedId=28073655&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-28073655

Reply:
Yes, they confirmed that we can not use `rhoai` registry, `modh` is been obsolete long time ago. 
**_Maybe_** as solution could be to use temporarily the build images from upstream. And ofc update again once 2.24 is live with the images from here: https://registry.redhat.io/rhoai